### ISSUE TITLE
web version

### ISSUE THREAD

**Issue by mgarcia-org** [javagl/gltfOverview#9]
Impressive, just wondering if there's a text or web page version of this?
Thanks

**Comment by javagl** [comment]
There currently are only the PNG and PDF files as releases. 

It's not entirely clear what you had in mind when you said "web page version". But I just naively dragged-and-dropped the SVG into the browser, and (with some tweaks to the page size) it is displayed surprisingly well. One could probably imagine embedding it on a website, with a some simple "SVG Viewer" wrapped around it, with more convenient zooming and panning functionality. 

(One step further, one could even consider some sort of links, maybe linking from the keywords to the spec or so, but this wouldn't be so simple any more...)

I'll definitely consider something like offering it as an interactive, zoomable SVG on GitHub pages. (But although it should not be sooo difficult, I cannot promise any timeline here...)

**Comment by mgarcia-org** [comment]
Hi, I just meant a html/html5 page, svg on my firefox (60) renders only A4 size, cropping the rest.
Or a print PDF version would also be great.
Cheers.

**Comment by javagl** [comment]
Yes, the cropping can be avoided easily by adjusting the page size (admittedly, I never cared about that, because it did not matter for the PNG and PDF, but I'll fix that). 

But even with the right page size, a more convenient zooming/panning functionality than the built-in one of the browser might be handy.

I'm not a DTP expert, though: In how far would a "print version" of the PDF differ from the current one? (Note that the reference card at https://www.khronos.org/developers/reference-cards/ is essentially the same as the overview here, with some additional Khronos branding stuff)

**Comment by emackey** [comment]
Maybe just add a link to the README?

https://www.khronos.org/files/gltf20-reference-guide.pdf
